makefile

```diff
@@ -27,7 +27,7 @@ uninstall: # PRIVATE
 	@rm -rf node_modules
 	@rm -rf ui/node_modules
 	@rm -rf dev/eslint-plugin-guardian-frontend/node_modules
-	@rm -rf tools/amp-validation/node_module
```

Comment:
Good catch 🔍

Reply:
```
adcock-lint has detected: 1 errors
```

Reply:
how come we don't want to delete this on a reinstall?

Reply:
nice!